packages/gitea/zarf.yaml

```
@@ -67,6 +67,13 @@ components:
           - gitea-values.yaml
     actions:
       onDeploy:
+        before:
+          - description: Check that the cluster has a storage class
+            maxTotalSeconds: 3
+            wait:
+              cluster:
+                kind: storageclass
+                name: "###ZARF_STORAGE_CLASS###" # TODO@lucasrod16: figure out why this doesn't work
```

```suggestion
                name: "${ZARF_STORAGE_CLASS}"
```
You need to use the actions syntax since in the end this will be pulled from the environment.